src/Fantomas.Core/CodePrinter.fs

```diff
@@ -2191,13 +2191,21 @@ let genKeepIdent (startNode: Node) (e: Expr) ctx =
         indentSepNlnUnindent (genExpr e) ctx
 
 let colGenericTypeParameters typeParameters =
-    coli sepComma typeParameters (fun idx t ->
-        let leadingSpace =
-            match t with
-            | Type.Var n when idx = 0 && n.Text.StartsWith("^") -> sepSpace
-            | _ -> sepNone
+    let short sep =
```

Comment:
Maybe rename to `genParameters` instead of `short`.
We could have a separate `let short = genParameters sepComma`.